DAO/UserDAO.py

```diff
@@ -40,12 +40,17 @@ def get_contacts(self, uid):
         return cursor.fetchall()
 
     def get_user_by_username(self, username):
+        """
+        Queries DB for information on given username
+        :param username: str
+        :return: RealDictCursor
+        """
         cursor = self.get_cursor()
-        query = 'SELECT uid, username, first_name, last_name, email, phone_number ' \
+        query = 'SELECT * ' \
```

Comment:
This would return the password del user, este metodo se usa en sitios para devolver el usuario en el response.

Reply:
password is hashed but you have a point, I'll use another method to just get the password internally